### ISSUE TITLE
Unable to set a sample rate

### ISSUE THREAD

**Issue by misterarias** [prezi/django-zipkin#12]
As it stands, I can manually force a trace to be logged *only* by manually sending an appropriate header; but I see no way to have a set sample rate. 
A generic one, defined at the middleware level, should be enough, although one could be able to set a different one for specific endpoints/views.

Thanks for the help, sorry if this is has an obvious answer.

**Comment by zsol** [comment]
You're right, the current implementation assumes someone else (upstream) makes the decision of triggering tracing.

It shouldn't be too hard to add, though ;)

**Comment by misterarias** [comment]
Alright, now that I know I'm not blind, I may be able to hack something =)

Thanks for that super-fast response 😄
